### PR TITLE
End point route

### PR DESCRIPTION
**Pull request by LouiaR**
Add end point route
     -----> End point route to  get all applicants
     -----> End point route to add new applicant
     ----->End point route to get one applicant
Add docs folder
    ------> Add a schema .png file 
Edit read me file 
    ------> Add schema title 
    ------> Display schema .png

### CODE REVIEW

File: backend/routes/applicants.js [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
+    res.status(200).json({
+      applicants: rows
+    });
+  });
+});
+
+
+// get one applicant through his id
+router.get('/:id', function (req, res) {
+  var sql = 'select * from applicants where id = ?';
```
Comment (OlenaKashuba) [inline backend/routes/applicants.js]: you might want to use let or const instead of var

Comment (LouiaR) [reply backend/routes/applicants.js]: Ok

File: backend/routes/applicants.js [hunk 2]
```diff
@@ -1,59 +1,59 @@
...
-		tel: '98483247282',
-		status: 'Asylum seeker',
-		country: 'Eritrea',
-		experience: 'very little',
-		itAcess: true,
-		hearAbout: 'website'
-	}]);
+	let sql = 'select * applicants';
+  db.all(sql, [], (err, rows) => {
+    console.log((rows.length))
```
Comment (OlenaKashuba) [inline backend/routes/applicants.js]: it is a good practice to delete all of the spare console.logs from commits

File: backend/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+To Run
+============
+1. `> yarn or npm` , install all dependency .
+2. `> yarn start  or npm start`, start server.
+
+
+
+Api end-point
+============
+2. `localhost:3002/api/applicants` , to get all the inserted data.
```
Comment (dahfool) [inline backend/README.md]: please update to 3001

**Pull request merged**